src/plugins/python/reporter_bugzilla.py

```diff
@@ -117,6 +117,8 @@ def set_settings(bz: Dict, settings: Dict):
         environ = settings.get('DontMatchComponents')
     bz['b_DontMatchComponents'] = environ or ''
 
+    if bz.get('b_create_private'):
+        bz['b_create_private'] = False
```

Comment:
“If `b_create_private` is already set, override it to `False`”? Is this intended?

Reply:
Ouch, no, of course not. I am missing "not" there. Good catch!

Reply:
https://github.com/abrt/libreport/pull/760